include/fastdds/rtps/participant/RTPSParticipant.h

```diff
@@ -283,6 +283,25 @@ class RTPS_DllAPI RTPSParticipant
     bool ignore_reader(
             const GUID_t& reader_guid);
 
+    /**
+     * Enables the monitor service in this RTPSParticipant.
+     *
+     * @return true if the monitor service could be correctly enabled.
+     *
+     * @note Not supported yet. Currently returns RETCODE_UNSUPPORTED
```

Comment:
```suggestion
     * @note Not supported yet. Currently always returns false
```

Please, apply to other RTPS methods.